README.md

```diff
@@ -12,27 +12,40 @@
 Habitat-Lab
 ==============================
 
-Habitat-Lab is a modular high-level library for end-to-end development in embodied AI --
-defining embodied AI tasks (e.g. navigation, rearrangement, instruction following, question answering), configuring embodied agents (physical form, sensors, capabilities), training these agents (via imitation or reinforcement learning, or no learning at all as in SensePlanAct pipelines), and benchmarking their performance on the defined tasks using standard metrics.
+Habitat-Lab is a modular high-level library for end-to-end development in embodied AI. It is designed to train agents to perform a wide variety of embodied AI tasks in indoor environments, as well as develop agents that can interact with humans in performing these tasks.
+
+Towards this goal, Habitat-Lab is designed to support the following features:
```

Comment:
Let's use the same capitalisation for all the features. 

In first 2 feature headings all words are capitalised. In 3rd feature heading "agents" is lowercase though in 2nd feature heading it is capitalised. In 4th feature heading only first word is capitalised.